data/src/main/java/com/globant/equattrocchio/data/ImagesServicesImpl.java

```diff
@@ -13,31 +14,30 @@
 
 public class ImagesServicesImpl implements ImagesServices {
 
-    private static final String URL= "http://splashbase.co/";
+    private static final String URL = "http://splashbase.co/";
 
     @Override
-    public void getLatestImages(Observer<Boolean> observer) {
+    public void getLatestImages(final Observer<String> observer) {
         Retrofit retrofit = new Retrofit.Builder().
                 baseUrl(URL).
                 addConverterFactory(GsonConverterFactory.create())
                 .build();
 
-        SplashbaseApi api  = retrofit.create(SplashbaseApi.class);
+        SplashBaseApi api = retrofit.create(SplashBaseApi.class);
 
         Call<Result> call = api.getImages();
 
         call.enqueue(new Callback<Result>() {
             @Override
             public void onResponse(Call<Result> call, Response<Result> response) {
-                //todo: show the response.body() on the ui
+                observer.onNext(new Gson().toJson(response.body()));
```

Comment:
this is a bad usage of observables.
Retrofit can actually return observables instead of calls and you should be returning an observable and subscribing to it in the top layers.
something like:
``` java 
Observable<Response<Result>> observable = api.getImages();
return observable
```